server/domains/conferences.py

```diff
@@ -0,0 +1,26 @@
+from uuid import uuid4
+
+from fastapi import APIRouter, WebSocket
+from fastapi.responses import JSONResponse, Response
+
+from server.conference import main_conference_manager
+from server.conference.exceptions import ConferenceNotFound
+
+
+router = APIRouter()
+
+
+@router.post("/conference/")
+async def post():
```

Comment:
Not really good naming. Should be renamed to `create_conference` or something

Reply:
Fixed